test/ci-operator-integration/expected_files/expected_with_template.json

```diff
@@ -0,0 +1,1533 @@
+{
```

Comment:
I'm surprised we are not seeing api group version and kind in these, so that makes me think we're using the wrong serializer